pkg/neutronapi/volumes.go

```diff
@@ -31,7 +31,14 @@ func GetVolumes(name string, extraVol []neutronv1beta1.NeutronExtraVolMounts, sv
 	}
 	for _, exv := range extraVol {
 		for _, vol := range exv.Propagate(svc) {
-			res = append(res, vol.Volumes...)
+			for _, v := range vol.Volumes {
+				volumeSource, _ := v.VolumeSource.ToCoreVolumeSource()
```

Comment:
It doesn't really matter, but the `VolumeSource` part of this is already implied, so it doesn't need to be explicitly mentioned. It could be:
```
v.ToCoreVolumeSource()
```

Reply:
Done